circle.yml

```
@@ -254,6 +254,10 @@ commands:
         description: Whether to use wait-on to wait on a server to be booted
         type: string
         default: ""
+      start-script:
+        description: Custom npm start script for repo
+        type: string
+        default: ""
```

```suggestion
        default: "npm start --if-present"
```
then you can remove `when` and `unless` conditions down in this file, since you can pass custom command